keras_cv/layers/preprocessing3d/global_z_rotation.py

```diff
@@ -0,0 +1,76 @@
+
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d.base_augmentation_layer_3d import (
+    BaseAugmentationLayer3D,
+)
+from keras_cv.ops.point_cloud import coordinate_transform, wrap_angle_rad
+
+
+POINT_CLOUDS = "point_clouds"
+BOUNDING_BOXES = "bounding_boxes"
+
+class GlobalZRotation(BaseAugmentationLayer3D):
```

Comment:
Are we going to standardize the random naming like KPL?
As we have already accumulated a bit of drift (see https://github.com/keras-team/keras-cv/issues/826)

Reply:
+1 -- we need to have a good convention here and this is drifting from our already weak convention in a big way.

Reply:
I changed the names to
POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES